bin/config.py

```diff
@@ -58,13 +58,83 @@
 
 # Below here is some global state that will be filled in main().
 
-# Some default values needed for tests.
 args = argparse.Namespace()
-args.testcases = None
-args.cpp_flags = None
-args.verbose = None
-args.force_build = None
-args.error = None
+
+default_args = {
+    'jobs': os.cpu_count() // 2,
+    'time': 600,
```

Comment:
Add a comment that this is for `fuzz` only (option names aren't very descriptive out of context, but that's ok)

Reply:
:+1: (57be5a8)